gyp/pylib/gyp/generator/make.py

```diff
@@ -1739,6 +1739,8 @@ def WriteTarget(
                         ldflags.append(r"-Wl,-rpath=\$$ORIGIN/")
                         ldflags.append(r"-Wl,-rpath-link=\$(builddir)/")
                 library_dirs = config.get("library_dirs", [])
+                if library_dirs:
+                    library_dirs = [Sourceify(self.Absolutify(i)) for i in library_dirs]
```

Comment:
Let's use an Assignment Expression...
```suggestion
                if library_dirs := config.get("library_dirs", []):
                    library_dirs = [Sourceify(self.Absolutify(i)) for i in library_dirs]
```